tutorials/incorporating_oli_calculations.ipynb

```diff
@@ -233,45 +299,90 @@
    },
    "outputs": [],
    "source": [
-    "if credential_manager:\n",
-    "    with OLIApi(credential_manager) as oliapi:\n",
-    "        # create a new DBS file\n",
-    "        dbs_file_id = oliapi.get_dbs_file_id(\n",
-    "            chemistry_source=source_water[\"components\"],\n",
-    "            phases=[\"liquid1\", \"solid\"],\n",
-    "        )\n",
-    "        \n",
-    "        # create water analysis inputs\n",
-    "        water_analysis_input = f.build_flash_calculation_input(\n",
-    "            \"wateranalysis\",\n",
-    "            state_vars,\n",
-    "        )\n",
-    "    \n",
-    "        # create water analysis case\n",
-    "        water_analysis_base_case = f.run_flash(\n",
-    "            \"wateranalysis\",\n",
-    "            oliapi,\n",
-    "            dbs_file_id,\n",
-    "            water_analysis_input,\n",
-    "            file_name=\"water_analysis_singlepoint\"\n",
-    "        )\n",
-    "        \n",
-    "        # create apparent composition from water analysis output\n",
-    "        water_analysis_apparent_composition = f.build_flash_calculation_input(\n",
-    "            \"isothermal\",\n",
-    "            state_vars,\n",
-    "            water_analysis_base_case[0],\n",
-    "        )\n",
-    "        \n",
-    "        # run isothermal flash for the survey parameters\n",
-    "        isothermal_survey_result = f.run_flash(\n",
-    "            \"isothermal\",\n",
-    "            oliapi,\n",
-    "            dbs_file_id,\n",
-    "            water_analysis_apparent_composition,\n",
-    "            survey,\n",
-    "            \"isothermal_composition_survey\",\n",
-    "        )"
+    "# Flash calculation example #1: Water Analysis survey with pH reconciliation.\n",
+    "\n",
+    "# OLIApi is used as a context manager\n",
+    "with OLIApi(credential_manager, interactive_mode=interactive_mode) as oliapi:\n",
```

Comment:
```suggestion
    "with OLIApi(credential_manager, interactive_mode=False) as oliapi:\n",
```
as is, this will produce an error since interactive_mode no longer exists.